apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -42,7 +42,7 @@ namespace Google.Cloud.Spanner.V1
     /// </remarks>
     public sealed class PooledSession : IDisposable
     {
-        private readonly Session _session;
+        internal Session Session { get; }
```

Comment:
Made it internal because now we are passing entire PooledSession object in ResultStream instead of directly this session instance. Therefore we now need to access this via PooledSession object in ResultStream class.